Source/WebInspectorUI/UserInterface/Views/FolderTreeElement.js

```diff
@@ -25,11 +25,36 @@
 
 WI.FolderTreeElement = class FolderTreeElement extends WI.GeneralTreeElement
 {
-    constructor(title, representedObject)
+    constructor(title, representedObject, id = null)
     {
         const classNames = [WI.FolderTreeElement.FolderIconStyleClassName];
         const subtitle = null;
         super(classNames, title, subtitle, representedObject, {hasChildren: true});
+
+        if (id) {
+            let settingName = "folder-tree-element-" + id;
+            const defaultValue = false;
+            this._expandedSetting = new WI.Setting(settingName, defaultValue);
```

Comment:
NIT: i know our normal style is to use `const` for things like this, but in this case it's pretty obvious that the second argument to `WI.Setting` is the default value so we normally just inline it
```js
if (options.id) {
    this._expandedSetting = new WI.Setting("folder-tree-element-" + options.id, false);
```